README.md

```diff
@@ -2,11 +2,20 @@
 
 create secure peer to peer networks using secret-handshakes.
 
-SecretStack is built on [secret-handshake](https://github.com/dominictarr/secret-handshake)
-and [muxrpc](https://github.com/ssbc/muxrpc). This provides a framework
-to make building secure, decentralized systems easier.
+This provides a framework to make building secure, decentralized systems easier.
 (such as [ssb-server](https://github.com/ssbc/ssb-server) which this was refactored out of ;)
 
+This module:
+
+* uses [secret-handshake](https://github.com/auditdrivencrypto/secret-handshake) to set up a shared key for use between two peers with public / private keys (and verify the other peer)
+* uses [multiserver](https://github.com/ssb-js/multiserver) to handle different ways of connecting to other peers over different protocols (who you then handshake with)
+* uses muxrpc to provide a remote process call (rpc) interface to peers who have authenticated and connected allowing them to call methods on each other (like "please give me all mix's messages since yesterday"). This is all encrypted with the shared key set up by secret handshake.
```

Comment:
Link to `muxrpc` here?

Reply:
good call I have added a link

Also I'm curious if these docs are accurate and helpful. It was just something I copied and pasted from a previous issue about docs

Reply:
Pretty accurate. The docs always look different anyway depending on who wrote them.